website/package.json

```diff
@@ -3,22 +3,37 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "docusaurus-start",
-    "build": "docusaurus-build",
-    "gh-pages": "docusaurus-publish",
-    "examples": "docusaurus-examples",
-    "crowdin-upload": "crowdin --config ../crowdin.yaml upload sources --auto-update -b master",
-    "crowdin-download": "crowdin --config ../crowdin.yaml download -b master",
-    "publish-gh-pages": "docusaurus-publish",
-    "write-translations": "docusaurus-write-translations",
-    "version": "docusaurus-version",
-    "rename-version": "docusaurus-rename-version"
+    "docusaurus": "docusaurus",
+    "start": "docusaurus start",
+    "build": "docusaurus build",
+    "swizzle": "docusaurus swizzle",
+    "deploy": "docusaurus deploy",
+    "serve": "docusaurus serve",
+    "crowdin:upload": "cd .. && crowdin upload sources --config crowdin-v2.yaml",
+    "crowdin:downloadTranslations": "cd .. && crowdin download --config crowdin-v2.yaml",
```

Comment:
`cd` is somewhat weird, IMO. Can you use `yarn workspace` instead?

Reply:
Usually I use "--cwd ..." but couldn't make this work with yarn 2.

It seems the following would work:

```
    "netlify:crowdin": "yarn workspace @jest/monorepo crowdin:upload && yarn workspace @jest/monorepo crowdin:download"
```

Note: the Crowdin cli behaves a bit weirdly regarding relative paths, it's better to put the yaml  file at the root and also execute the cli from the root in my experience

Reply:
> Usually I use "--cwd ..." but couldn't make this work with yarn 2.

It's a bug https://github.com/yarnpkg/berry/issues/2346, you can work around it with `yarn ../ crowdin download --config crowdin-v2.yaml`

Reply:
thanks, didn't know.

Will keep the existing solution anyway as both solutions work